tobac/tests/test_utils_mask.py

```diff
@@ -0,0 +1,1382 @@
+            )
+        ),
+        attrs=dict(units="feature"),
+    )
+
+    test_features = pd.DataFrame(
+        {
+            "feature": [1, 2, 3],
+            "frame": [0, 1, 2],
+            "time": pd.date_range(
+                datetime(2000, 1, 1, 0), datetime(2000, 1, 1, 2), periods=3
+            ),
+            "cell": [1, 1, -1],
+        }
+    )
+
+    mask_copy = test_mask.copy(deep=True)
+    features_copy = test_features.copy(deep=True)
+
+    feature_mask = convert_cell_mask_to_features(test_features, test_mask, stubs=-1)
+
+    # Test dataframe is the same
+    pd.testing.assert_frame_equal(test_features, features_copy)
+
+    # Test mask is the same
+    assert mask_copy.equals(test_mask)
+
+
+def test_convert_cell_mask_to_features_inplace():
+    """Test that convert_cell_mask_to_features does alter the input mask when
+    the inplace keyword is used
+    """
+    test_data = np.zeros([3, 4, 5], dtype=int)
+    test_data[0, 1:3, 1:4] = 1
+    test_data[1, 1:3, 1:4] = 1
+    test_data[2, 1:3, 1:4] = -1
+
+    test_mask = xr.DataArray(
+        test_data,
+        dims=("time", "y", "x"),
+        coords=dict(
+            time=pd.date_range(
+                datetime(2000, 1, 1, 0), datetime(2000, 1, 1, 2), periods=3
+            )
+        ),
+        attrs=dict(units="feature"),
+    )
+
+    test_features = pd.DataFrame(
+        {
+            "feature": [1, 2, 3],
+            "frame": [0, 1, 2],
+            "time": pd.date_range(
+                datetime(2000, 1, 1, 0), datetime(2000, 1, 1, 2), periods=3
+            ),
+            "cell": [1, 1, -1],
+        }
+    )
+
+    mask_copy = test_mask.copy(deep=True)
+    features_copy = test_features.copy(deep=True)
+
+    feature_mask = convert_cell_mask_to_features(
+        test_features, test_mask, stubs=-1, inplace=True
+    )
+
+    # Test dataframe is the same
+    pd.testing.assert_frame_equal(test_features, features_copy)
+
+    # Test mask is the same
+    assert feature_mask.equals(test_mask)
+    assert not mask_copy.equals(test_mask)
+
+
+def test_convert_feature_mask_to_tracks_single_track():
+    """Test basic functionality of convert_feature_mask_to_tracks with a single
+    track
+    """
+    test_data = np.zeros([3, 4, 5], dtype=int)
+    test_data[0, 1:3, 1:4] = 1
+    test_data[1, 1:3, 1:4] = 2
+    test_data[2, 1:3, 1:4] = 3
+
+    test_mask = xr.DataArray(
+        test_data,
+        dims=("time", "y", "x"),
+        coords=dict(
+            time=pd.date_range(
+                datetime(2000, 1, 1, 0), datetime(2000, 1, 1, 2), periods=3
+            )
+        ),
+        attrs=dict(units="feature"),
+    )
+
+    test_features = pd.DataFrame(
+        {
+            "feature": [1, 2, 3],
+            "frame": [0, 1, 2],
+            "time": pd.date_range(
+                datetime(2000, 1, 1, 0), datetime(2000, 1, 1, 2), periods=3
+            ),
+            "cell": [1, 1, 1],
+            "track": [1, 1, 1],
```

Comment:
The functions `convert_feature_mask_to_track` and `convert_cell_mask_to_track` require the format of the original `Features` pandas dataframe with an added column `track` it seems like. As in one of your example notebooks (MCS tracking in ICON simulations), the suggested workflow is: 

```
merge_splits = merge_splits.merge_split_MEST(Track, dxy = dxy)
Track['track'] = merge_splits.feature_parent_track_id
```

which I think is good but it needs to be documented somewhere. From a user perspective, it is not clear whether to input the output of `merge_splits.merge_split_MEST` or do that extra step. I am making this comment here, because maybe we should include this in the test as well; to ensure the output format of merging and splitting can be used with this function.